### ISSUE TITLE
Mismatch error installing version 4.14 of OpenRCT2

### ISSUE THREAD

**Issue by jcurbaniak** [OpenRCT2/OpenRCT2#22986]
### Operating System

Windows 10 64 bit

### OpenRCT2 build

OpenRCT2, v0.4.14 (18492da) provided by GitHub  Did not update to 4.15

### Base game

RollerCoaster Tycoon 2

### Describe the issue

Install of 4.15 version gets mismatch error.

### Steps to reproduce

1. Attempt to install version 4.15

### Attachments

![Image](https://github.com/user-attachments/assets/dd79fa00-1e91-4d6b-885a-6eb57ec385ba)

**Comment by jcurbaniak** [comment]
Also get spelling errors in text messages.

**Comment by sjoerddebruin** [comment]
Common problem, please replace the data folder of your install using the portable download or reinstall OpenRCT2.

**Comment by jcurbaniak** [comment]
Re-installed OpenRCT2 and no mismatch this time.

 

But now getting “Load Game Failed to load …  File contains invalid data!”

 

Jerry U.

**Comment by jcurbaniak** [comment]
These are all from 4.14 version.   I have not saved any parks under 4.15.

 

Looks like I will fall back to 4.14.

 

Jerry U.

**Comment by 733737** [comment]
that shouldn't happen. It makes no sense to stay on an older version of the game indefinitely. You should update at some point. It is possible your save game is corrupted in some way.

**Comment by jcurbaniak** [comment]
Everything was fine under 4.14.  Now that I have installed 4.15 I cannot load saved games.

 

I have many hours spent buildings my parks and I do want to lose all that.

 

Anyway to make 4.15 conpatible with 4.14 saves?

 

I agree with your point about staying on an old version.

 

Jerry U.

**Comment by jcurbaniak** [comment]
I rolled OpenRCT2 back to 4.14 and once again I can load my saved parks.

 

Please fix this with 4.15 if you can.   I can send a sample saved park if that would help.

 

Jerry U.

 

From: ***@***.*** ***@***.***> 
Sent: Monday, October 14, 2024 5:41 PM
To: 'OpenRCT2/OpenRCT2' ***@***.***>
Subject: RE: [OpenRCT2/OpenRCT2] Mismatch error installing version 4.14 of OpenRCT2 (Issue #22986)

 

Everything was fine under 4.14.  Now that I have installed 4.15 I cannot load saved games.

 

I have many hours spent buildings my parks and I do want to lose all that.

 

Anyway to make 4.15 conpatible with 4.14 saves?

 

I agree with your point about staying on an old version.

 

Jerry U.

**Comment by Gymnasiast** [comment]
You didn’t install OpenRCT2 v0.4.15 correctly. Properly uninstall the old version and reinstall it.

Also, please start putting your stuff under Discussions/Q&A, rather than filing bug reports. Apart from one issue, every single report you made has turned out to be things working as intended, or errors on your end.

**Comment by jcurbaniak** [comment]
I will put it where I want.

**Comment by jcurbaniak** [comment]
I never knew the old release had to uninstalled first.  This is not stated anywhere as far as I know.

 

I have in the past done an update in place with no issues.

 

Release 4.15 “broke” the load of saved games from 4.14 and apparently is not going to be fixed.

 

I know this for a fact as when I regressed back to 4.14 I was able to load saved games again.

 

So it appears I will stay on 4.14 indefinitely as I have many hours spent in building my parks.

 

How the heck do I know what is “intended”?   If something does not appear logical to me that appears to me to be a bug.

 

Jerry U.
